ui/src/app/workflows/components/workflow-logs-viewer/workflow-logs-viewer.tsx

```diff
@@ -297,8 +297,9 @@ export function WorkflowLogsViewer({workflow, nodeId, initialPodName, container,
                 )}
                 {execSpec(workflow).podGC && (
                     <>
-                        <WarningIcon /> Your pod GC settings will delete pods and their logs immediately on completion.
+                        <WarningIcon /> Your pod GC settings will delete pods and their logs {execSpec(workflow).podGC.deleteDelayDuration ? "after ${execSpec(workflow).podGC.deleteDelayDuration}" : "immediately"} on completion.
```

Comment:
```suggestion
                        <WarningIcon /> Your pod GC settings will delete pods and their logs {execSpec(workflow).podGC.deleteDelayDuration ? `after ${execSpec(workflow).podGC.deleteDelayDuration}` : "immediately"} on completion.
```

I initially forgot that ES6+ string interpolation use backticks; I actually modified my original suggestion almost immediately after I wrote it, but it seems like you applied it before the edit came through.
pretty uncommon that work on GH is synchronous instead of asynchronous 😅